knowledge-portal/source/.gitlab-ci.yml

```diff
@@ -0,0 +1,51 @@
+copy-files:
+  stage: copy_to_central_repo
```

Comment:
The stage name would again be `push`, and the job name(L1) can be `copy-to-central-git` 🙂